code/datums/components/tackle.dm

```diff
@@ -233,15 +262,111 @@
 				user.SetKnockdown(0, ignore_canstun = TRUE)
 				user.get_up(TRUE)
 				user.forceMove(get_turf(target))
-				target.adjustStaminaLoss(40)
+				target.apply_damage(60, STAMINA)
 				target.Paralyze(0.5 SECONDS)
 				target.Knockdown(3 SECONDS)
 				if(ishuman(target) && ishuman(user))
-					INVOKE_ASYNC(S.dna.species, TYPE_PROC_REF(/datum/species, grab), S, T)
-					S.setGrabState(GRAB_AGGRESSIVE)
+					INVOKE_ASYNC(human_sacker.dna.species, TYPE_PROC_REF(/datum/species, grab), human_sacker, human_target)
+					human_sacker.setGrabState(GRAB_AGGRESSIVE)
+					if(!human_target.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+						human_target.add_movespeed_modifier(/datum/movespeed_modifier/shove)
+						addtimer(CALLBACK(human_target, TYPE_PROC_REF(/mob/living/carbon, clear_shove_slowdown)), SHOVE_SLOWDOWN_LENGTH * 2)
 
+/**
+ * Our neutral tackling outcome.
+ *
+ * Our tackler and our target both get knocked off-balance. The target longer than the tackler. However, the tackler stands up after this outcome. This is maybe less neutral than it appears, but the tackler initiated, so...
+ * This outcome also occurs when our target has blocked the tackle in some way, preventing situations where someone tackling into a blocker is too severely punished as a result. Hence, this has its own proc.
+*/
 
-	return COMPONENT_MOVABLE_IMPACT_FLIP_HITPUSH
+/datum/component/tackler/proc/neutral_outcome(mob/living/carbon/user, mob/living/carbon/target, roll = 1, tackle_word = "tackle")
+
+	var/mob/living/carbon/human/human_target = target
+	var/mob/living/carbon/human/human_sacker = user
+
+	user.visible_message(span_warning("[user] lands a [tackle_word] on [target], briefly knocking both off-balance!"), span_userdanger("You land a [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!"), ignored_mobs = target)
+	to_chat(target, span_userdanger("[user] lands a [tackle_word] on you, briefly knocking you off-balance!"))
+
+	user.get_up(TRUE)
+	if(ishuman(user) && !human_sacker.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+		human_sacker.add_movespeed_modifier(/datum/movespeed_modifier/shove)
+		addtimer(CALLBACK(human_sacker, TYPE_PROC_REF(/mob/living/carbon, clear_shove_slowdown)), SHOVE_SLOWDOWN_LENGTH)
+
+	if(ishuman(target) && !human_target.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+		human_target.add_movespeed_modifier(/datum/movespeed_modifier/shove)
+		addtimer(CALLBACK(human_target, TYPE_PROC_REF(/mob/living/carbon, clear_shove_slowdown)), SHOVE_SLOWDOWN_LENGTH * 2) //okay maybe slightly good for the sacker, it's a mild benefit okay?
```

Comment:
We should probably just use status effects for this to stop you from copying the same line in so many places

Reply:
I suppose now is the best time to actually do that.